setup.py

```diff
@@ -238,7 +238,6 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
 ]
 dask = [
     'cloudpickle>=1.4.1, <1.5.0',
-    'dask<2021.3.1;python_version<"3.7"',  # dask stopped supporting python 3.6 in 2021.3.1 version
     'dask>=2.9.0, <2021.6.1;python_version>="3.7"',  # dask 2021.6.1 does not work with `distributed`
```

Comment:
Do we still need the `python_version>="3.7"` here?

Reply:
We don't  :)